views/reviews/edit.ejs

```diff
@@ -2,69 +2,70 @@
 
 <div class="container">
   <div class="row">
-    <h1 class="form-heading">Change Review</h1>
-
-    <form
-      class="form"
-      action="/campgrounds/<%= campground_id %>/reviews/<%= review._id %>?_method=PUT"
-      method="POST"
-    >
-      <div class="form-group">
-        <label for="rating">Leave a rating:</label>
-        <fieldset class="starability-basic form-group" id="rating">
-          <div>
-            <input type="radio" id="first-rate5" name="review[rating]" value="5"
-            <% if (review.rating === 5) { %> checked <% } %> />
-            <label for="first-rate5" title="5 stars - Amazing">
-              <%- '<span class="fa fa-star checked"></span>'.repeat(5) %>
-            </label>
-          </div>
-          <div>
-            <input type="radio" id="first-rate4" name="review[rating]" value="4"
-            <% if (review.rating === 4) { %> checked <% } %> />
-            <label for="first-rate4" title="4 stars - Very good">
-              <%- '<span class="fa fa-star checked"></span>'.repeat(4) %>
-            </label>
-          </div>
-          <div>
-            <input type="radio" id="first-rate3" name="review[rating]" value="3"
-            <% if (review.rating === 3) { %> checked <% } %> />
-            <label for="first-rate3" title="3 stars - Average">
-              <%- '<span class="fa fa-star checked"></span>'.repeat(3) %>
-            </label>
-          </div>
-          <div>
-            <input type="radio" id="first-rate2" name="review[rating]" value="2"
-            <% if (review.rating === 2) { %> checked <% } %> />
-            <label for="first-rate2" title="2 stars - Not good">
-              <%- '<span class="fa fa-star checked"></span>'.repeat(2) %>
-            </label>
-          </div>
-          <div>
-            <input type="radio" id="first-rate1" name="review[rating]" value="1"
-            <% if (review.rating === 1) { %> checked <% } %> />
-            <label for="first-rate1" title="1 star - Terrible">
-              <%- '<span class="fa fa-star checked"></span>'.repeat(1) %>
-            </label>
-          </div>
-        </fieldset>
-      </div>
-      <div class="form-group">
-        <textarea
-          class="form-control"
-          type="text"
-          name="review[text]"
-          placeholder="Write a review for this review..."
-          rows="5"
-        >
+    <div class="col">
+      <h1 class="form-heading">Change Review</h1>
+      <form
+        class="form"
+        action="/campgrounds/<%= campground_id %>/reviews/<%= review._id %>?_method=PUT"
+        method="POST"
+      >
+        <div class="form-group">
+          <h2 class="change-rating-h2">Leave a rating:</h2>
```

Comment:
I'd recommend to not use `h2` in a class name. `rating-selection-label` or something similar as a class name would make more sense.